analytics/src/main/java/com/rakuten/tech/mobile/analytics/Event.kt

```diff
@@ -0,0 +1,14 @@
+package com.rakuten.tech.mobile.analytics
+
+/**
+ * An event to be tracked.
+ * @param name Event's name
+ * @param parameters Event parameters
+ */
+class Event(name: String, parameters: Map<String, Any>) {
+
+    /**
+     * Method to track an event.
+     */
+    fun track() {}
```

Comment:
Please fix the [code smells](https://sonarcloud.io/project/issues?id=rakutentech_android-inappmessaging&pullRequest=133&resolved=false&types=CODE_SMELL) here.

Reply:
Done